scripts/cgroups/run_cgroup_limited_benchmark.py

```diff
@@ -0,0 +1,54 @@
+import os
+import subprocess
+import time
+import signal
+
+GB = 1024 * 1024 * 1024
```

Comment:
1024 is GiB. 1000 is GB.

Reply:
I know, but I've seen it used somewhat interchangeably. But you would argue to be precise here?

Reply:
Naming the variable `GiB` would satisfy me. :)